rca/project_styleguide/templates/patterns/base_page.html

```diff
@@ -2,27 +2,10 @@
 
 {% load static wagtailuserbar wagtailcore_tags wagtailimages_tags navigation_tags util_tags %}
 
-{% block meta_tags %}
-    {% if GOOGLE_TAG_MANAGER_ID %}
-        {# To enable GTM code you need to specify GOOGLE_TAG_MANAGER_ID in Heroku settings #}
-        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','{{ GOOGLE_TAG_MANAGER_ID|escapejs }}');</script>
-    {% endif %}
-
-    {# favicons #}
-    <link rel="apple-touch-icon" sizes="180x180" href="{% static "images/apple-touch-icon.png" %}">
-    <link rel="icon" type="image/png" sizes="32x32" href="{% static "images/favicon-32x32.png" %}">
-    <link rel="icon" type="image/png" sizes="16x16" href="{% static "images/favicon-16x16.png" %}">
-    <link rel="manifest" href="{% static "images/site.webmanifest" %}">
-    <link rel="mask-icon" href="{% static "images/safari-pinned-tab.svg" %}" color="#000000">
-    <link rel="shortcut icon" href="{% static "images/favicon.ico" %}">
-    <meta name="msapplication-TileColor" content="#000000">
-    <meta name="msapplication-config" content="{% static "images/browserconfig.xml" %}">
-    <meta name="theme-color" content="#000000">
```

Comment:
**food for though/ramblings**

I guess this is what prompted your recent comment about better use of base and base_page templates. I think the caveat is that we don't usually rely on base.html ro render elements with no `page` context and that's why we see it so ofter - I've never understood it tbh but I think your change does make sense, but I don't know if favicons belong in `base.html`, would we ever want to render them on something that's _not_ a page? Again it's probably moot because we don't use templates that way

Reply:
> I guess this is what prompted your recent comment about better use of base and base_page templates.

Yes, but it did bug me on a couple of projects prior to this - it's just something I didn't really have time to address before :)

>  but I don't know if favicons belong in base.html, would we ever want to render them on something that's not a page? Again it's probably moot because we don't use templates that way

I would say so, yes. It's important for all HTML responses to retain the original branding - which favicons are a part of.